src/text/TextBody.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react';
+
```

Comment:
Why `src/text`?

Reply:
Oh I see, you are trying to create an internal Design System to use for building all those pieces.

got it!

Reply:
We could also do something like `src/internal/text`